examples/svgo-export/src/svgo-export.js

```diff
@@ -39,12 +42,12 @@ export function onExportSlices(context) {
 
     // Now we run through each one...
     paths.forEach(path => {
-      log(`Compressing SVG files in ${path}`)
+      console.log(`Compressing SVG files in ${path}`)
```

Comment:
I don't think this works in Sketch < 50? Is https://github.com/skpm/console required?

Reply:
it actually might be nice to switch these to `sketch/ui` anyway :)

Reply:
heh ok, polyfilling might be nice rather than confusing people on Sketch < 50